src/passthrough/mod.rs

```diff
@@ -1479,4 +1483,144 @@ mod tests {
         assert!(!fs.cfg.no_open);
         assert!(!fs.cfg.writeback);
     }
+
+    impl ZeroCopyReader for File {
```

Comment:
Why are these needed? The commit message does not seem to explain it.

Reply:
This resides in `mod test`, at https://github.com/cloud-hypervisor/fuse-backend-rs/pull/184/files/9ca710da29796b759d6d782e8dd2d9209d5bf838#diff-7f83e7da9ae6f4daf23fadb16185e191349a8f96620bd58d29483e3b28ec1a1cR1593

I need a `&mut dyn ZeroCopyReader`, in fuse-backend-rs, I didn't find a convenient implementation of `ZeroCopyReader` and `ZeroCopyWriter`, so I have to implement one in `mod test` for test usage.